src/main/flight/mixer.c

```diff
@@ -793,7 +759,7 @@ float applyThrottleLimit(float throttle) {
 
 void mixWithThrottleLegacy(float *motorMix, float controllerMixMin, float controllerMixMax) {
     // clipping adjustment
-    if (controllerMixRange > 1.0f && (hardwareMotorType != MOTOR_BRUSHED)) {
```

Comment:
why was this removed?

Reply:
because if we don't normalize we obtain negative values that cannot be linearized (sqrt).

But other that that, considering  motorOutput = motorOutputMin + motorMix[i] * motorOutputRange, without normalization if the mixer has a value > 1 (we can have up to +-3 if I'm not wrong) this would exceed the motorOutputRange. The scope to have a normalized MotorMix is to distribute controller/throttle decisions over the full range of operation of the motors. Without normalization I think that the values > 1 or < 0 would just be clipped.

Reply:
yes correct values above and below 1 and 0 are just clipped. However this line of code was added because of a lot of testing that silverware developers made when it comes to brushed motors. They found that brushed motors react so much slower than their brushless counterparts that just clipping without scaling the outputs leads to brushed motors feeling as though they have more power. Since the brushed motors act slow you don't notice the imbalance that not normalizing the outputs creates because they don't react fast enough for that to become an issue.  Hence for brushed whoops (the only place brushed is still used) you can get a whoop that feels like it has more power and can thus go around the track faster. This has been tested and proven by pilots such as project mockingbird.

Reply:
I understand, so can I constrain values for brushed motors to avoid negative values? And to make explicit that values beyond 0-1 are just clipped? I think it's better to guarantee that mixer's value are always in the 0/1 range because all the logic works with that assumption.